apps/frontend/src/modules/chat/chat-api.ts

```diff
@@ -0,0 +1,71 @@
+import { APIPath, ContentType } from "~/libs/enums/enums.js";
+import { BaseHTTPApi } from "~/libs/modules/api/api.js";
+import { type HTTP } from "~/libs/modules/http/http.js";
+import { type Storage } from "~/libs/modules/storage/storage.js";
+
+import { AIAssistantApiPath } from "./libs/enums/enums.js";
+import {
+	type AIAssistantCreateMultipleTasksDto,
+	type AIAssistantResponseDto,
+	type AIAssistantSuggestTaskRequestDto,
+} from "./libs/types/types.js";
+
+type Constructor = {
+	baseUrl: string;
+	http: HTTP;
+	storage: Storage;
+};
```

Comment:
We have this type in the shared folder - **type APIConfiguration**
Can we use it here (and wherever else where it is re-created)?

Reply:
Not fixed